src/app/private/[uid]/admin/home/chapters/[chapterId]/page.tsx

```diff
@@ -47,7 +47,7 @@ const ChapterPage = async ({ params }: ChapterPageParams) => {
           </div>
           <div className="flex flex-row text-start">
```

Comment:
I think it would be nice if we can extract the card to a component. Since `ChapterLeader` and `User` also has this view.